source_test.go

```diff
@@ -60,3 +68,108 @@ func TestSource_Open_WithPosition(t *testing.T) {
 	is.NoErr(err)
 	is.True(pos.LastEditedTime.Equal(underTest.lastMinuteRead))
 }
+
+func TestSource_Configure(t *testing.T) {
+	tests := []struct {
+		desc   string
+		input  map[string]string
+		output Config
+		err    error
+	}{
+		{
+			desc: "Succeed without override",
+			input: map[string]string{
+				Token: "abc-def",
+			},
+			output: Config{token: "abc-def", pollInterval: time.Minute},
+		},
+		{
+			desc: "Succeed with override",
+			input: map[string]string{
+				Token:        "abc-def",
+				PollInterval: "2m",
+			},
+			output: Config{token: "abc-def", pollInterval: 2 * time.Minute},
+		},
+		{
+			desc: "Fail to override poll interval because too small",
+			input: map[string]string{
+				Token:        "abc-def",
+				PollInterval: "1s",
+			},
+			err: fmt.Errorf("poll interval must not be shorter than a minute (provided: 1s)"),
```

Comment:
There's already test coverage for this here: https://github.com/conduitio-labs/conduit-connector-notion/blob/main/config_test.go#L58

Reply:
interesting